js/activity.js

```diff
@@ -1514,6 +1514,7 @@ class Activity {
                     else if (ele.label === "Disable horizontal scrolling")
                         ele.display = true;
                 })
+                activity.textMsg(_("Horizontal scrolling enabled."),3000);
```

Comment:
please add a space after the ,

Reply:
I was not clear.

Please change:
                activity.textMsg(_("Horizontal scrolling enabled."),3000);
to:
                activity.textMsg(_("Horizontal scrolling enabled."), 3000);